RearrangementExceptions.java

```diff
@@ -0,0 +1,52 @@
+package CodeU_Assignment6;
+
+class NullRearrangementsException extends RearrangementExceptions{
+
+    /**
+     * serialVersionUID = 6995452535841683530L
+     */
+    private static final long serialVersionUID = 6995452535841683530L;

```

Comment:
Indeed, I have never payed attention that the Exception class was Serializable. Thanks for sharing this with me.